manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "dapp-addresses",
   "description": "Dapp for Parity",
   "author": "Parity Team <admin@parity.io>",
-  "version": "1.99.99",
+  "version": "1.0.0",
```

Comment:
Probably 2.0.0 (just to align with packge.json). We should probably have a CI job that updates these on building. (Then we also don't need to check in the compiled css & js files manually, CI can just build it and update).

Will get cracking on that - for now, just update the number to align and we should be fine.